### ISSUE TITLE
Duplicate server_name {{ DOMAINLIST }}

### ISSUE THREAD

**Issue by TaiSHiNet** [AnoopAlias/nDeploy#282]
On template server.j2 there is a duplicate block for Proxy Subdomains, causing a duplicate server_name and triggering warnings

**Comment by AnoopAlias** [comment]
This is intended. Nginx best practice says to use different server block for handling redirections to https etc rather than using if. Because we create the domain list from cpanel userdata ,the list will have duplicate entries . But nginx only takes the first server{} block containing the name and throw a warning that the rest is duplicate .The first server block that has the server name does the job.

In short, you can safely ignore the duplicate server name warnings at reloading etc as this is a side effect of confirming to nginx best practices

**Comment by TaiSHiNet** [comment]
I understand and agree that a separate server block + redirect is the right way, but we should handle the duplicates as well, maybe another if block? (and remove the dupes)

**Comment by AnoopAlias** [comment]
@TaiSHiNet  - I think the code we add for this will just be clutter as nginx loads the config on start/reload in-memory and works fine with it . the duplicates as the warning states will be ignored and having them there doesn't slow down nginx in any way. But having more code and regex matches in the config generation will definitely slow down that .In short we are simply wasting time to make nginx just not print a warning

**Comment by AnoopAlias** [comment]
@TaiSHiNet - I am closing this. But as mentioned any PR is always welcome
